app/src/main/kotlin/com/bitwarden/authenticator/ui/authenticator/feature/manualcodeentry/SaveManualCodeButtons.kt

```diff
@@ -0,0 +1,84 @@
+package com.bitwarden.authenticator.ui.authenticator.feature.manualcodeentry
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.semantics.testTag
+import androidx.compose.ui.unit.dp
+import com.bitwarden.authenticator.R
+import com.bitwarden.authenticator.ui.platform.components.button.BitwardenFilledButton
+import com.bitwarden.authenticator.ui.platform.components.button.BitwardenFilledTonalButton
+import com.bitwarden.authenticator.ui.platform.components.button.BitwardenOutlinedButton
+
+/**
+ * Displays save buttons for saving a manually entered code.
+ *
+ * @param state State of the buttons to show.
+ * @param onSaveLocallyClick Callback invoked when the user clicks save locally.
+ * @param onSaveToBitwardenClick Callback invoked when the user clicks save to Bitwarden.
+ */
+@Composable
+fun SaveManualCodeButtons(
+    state: ManualCodeEntryState.ButtonState,
+    onSaveLocallyClick: () -> Unit,
+    onSaveToBitwardenClick: () -> Unit,
+) {
+
+    when (state) {
+        ManualCodeEntryState.ButtonState.LocalOnly -> {
+            BitwardenFilledTonalButton(
+                label = stringResource(id = R.string.add_code),
+                onClick = onSaveLocallyClick,
+                modifier = Modifier
+                    .semantics { testTag = "AddCodeButton" }
```

Comment:
So I left this test tag here for the existing case. Do you know if there are tags I should add for the other buttons?

Reply:
I'm not sure what test tags are expected. I'd leave those up to the QA team to add for now.

Reply:
Cool 👍